lib/tictactoe.rb

```diff
@@ -1,58 +1,82 @@
 # frozen_string_literal: true
 
 class TicTacToe
-  def initialize(io:)
-    @board = Array.new(9, ' ')
+  def initialize(io:, board:)
+    @board = board
     @current_token = 'X'
     @io = io
   end
 
   def render_board
```

Comment:
I wonder if `render_board` might hew more to the role of `MainIO` in interfacing with the player moreso than to the game logic in `TicTacToe`.